src/garage/torch/algos/ppo.py

```diff
@@ -40,6 +41,8 @@ class PPO(VPG):
             dense entropy to the reward for each time step. 'regularized' adds
             the mean entropy to the surrogate objective. See
             https://arxiv.org/abs/1805.00909 for more details.
+        minibatch_size (int): Batch size for optimization.
+        optimization_epochs (int): Maximum number of epochs for update.
```

Comment:
please use `max_optimization_epochs` because minibatching can use early stopping in the future. my apologies for the back-and-forth.